Projections/Benchmark/Benchmark.csproj

```diff
@@ -25,48 +13,8 @@
     <ProjectReference Include="..\Windows\Windows.csproj" />
   </ItemGroup>
 
-  <Target Name="GenerateProjection" Condition="'$(GenerateTestProjection)' == 'true'">
-    <ItemGroup>
-      <!--Do not publish projection source winmds -->
-      <ReferenceCopyLocalPaths Remove="@(ReferenceCopyLocalPaths)" Condition="%(ReferenceCopyLocalPaths.Filename) == 'BenchmarkComponent'" />
-    </ItemGroup>
-    <PropertyGroup>
-      <CsWinRTVerbosity>high</CsWinRTVerbosity>
-      <CsWinRTResponseFile>$(GeneratedFilesDir)cswinrt_benchmark.rsp</CsWinRTResponseFile>
-      <CsWinRTCommand>$(CsWinRTExe) %40"$(CsWinRTResponseFile)"</CsWinRTCommand>
-    </PropertyGroup>
-    <PropertyGroup>
-      <CsWinRTParams>
--verbose
--target $(TargetFramework)
--in 10.0.18362.0
--in @(ReferenceWinMDs->'"%(FullPath)"', ' ')
--out "$(GeneratedFilesDir.TrimEnd('\'))"
--exclude Windows
```

Comment:
I see we have a \<CsWinRTIncludes\> below, do we have/need a \<CsWinRTExcludes\>Windows\</CsWinRTExcludes\> below as well?

Reply:
no, this demonstrates the default for CsWinRTExcludes of Windows;Microsoft